documentation/source/parameterisations.rst

```diff
@@ -1,5 +1,5 @@
-Parametrisations
-================
+Parameterisations
```

Comment:
Fair, we are rather inconsistent in how we spell parameterisation, it turns out. I think either is valid but we can pick and stick on this one.

Reply:
Raised #579